bin/sl-run.js

```diff
@@ -24,6 +24,9 @@ var agentOptions = {
   logger: config.logger,
   strongTracer: tracer(),
 };
+if (process.env.SPEED_UP_INTERVAL_FOR_TESTS === 'yes') {
```

Comment:
how about just making a `STRONGLOOP_BASE_INTERVAL` env var, and supporting it at: https://github.com/strongloop/strongops/blob/master/lib/config.js#L5

I think we should STRONGLOOP prefix our env vars, even if they  are sentences, and putting it in lib/config.js makes it easy to find, and useful to agent tests, etc.